requirements.txt

```diff
@@ -23,13 +23,9 @@ cryptography==2.6.1       # via pyopenssl, service-identity
 cssselect==1.0.3          # via parsel, premailer, scrapy
 cssutils==1.0.2           # via premailer
 docutils==0.14            # via awscli, botocore
-enum34==1.1.6             # via cryptography
```

Comment:
Have you used python 2 when running pip-compile? Curious, because for me it always adds the aux dependencies

Reply:
I didn't. Should I re-run it using python2 instead?

Reply:
Yeah, as far as the stack is based on python:2, we need to use python 2 to compile the deps

Reply:
Updated. It's now pulling in setuptools though :confused:

Reply:
Thanks! Well, we lived fine w/o setuptools in requirements, I guess it's considered an anti-pattern now. Feel free to drop it, and let's merge it

Reply:
Btw what pip version do you use?

Reply:
19.1.1. I'll downgrade to an older one and see if that removes it. If not I'll just manually remove it

Reply:
Thanks. I see, I used an old version of pip-tools (3.3.2), and after upgrading to the latest 3.8.0 it adds setuptools to the requirements.txt

Reply:
I downgraded pip-tools from 3.8.0 to 3.7.0 and it removed setuptools as a dependency

Reply:
Now, after all, I see my comments about setuptools as nitpicking, apologizes: it seems that as we do a fresh install, there's no harm in specifying setuptools explicitly, especially as it's added by pip-tools automatically. We may revise it in the future, and hopefully forget about it after migration to python 3. Merged :+1: